cirq/protocols/json_test_data/BitstringAccumulator.repr

```diff
@@ -0,0 +1,14 @@
+cirq.BitstringAccumulator(
+    meas_spec=cirq.MeasurementSpec(
+        max_setting=cirq.InitObsSetting(
+            init_state=cirq.PLUS_Z(cirq.GridQubit(5, 2)) * cirq.MINUS_X(cirq.GridQubit(7, 8)),
+            observable=cirq.Z(cirq.GridQubit(5, 2)) * 1),
```

Comment:
The `* 1` here is a bit odd. Just the repr unconditionally including it I guess.

Reply:
There are several dirty things going on here.

1. Multiplying by 1 turns a `Z` gate into a `PauliString`, which `observable` must be
2. This isn't actually the result of calling `repr` on `BitstringAccumulator`. Rather, this is crafted to create an object that passes the serialization tests. I don't think there's anything that makes sure the saved `.repr` file matches the output from `repr`. 

Of course, expecting the repr to be a complete serialization of such a rich bag-of-data class such as this is a bit too much to ask. How do other classes initialized by numpy arrays work?